apps/backend/src/scripts/sftp.bcm.sh

```diff
@@ -0,0 +1,49 @@
+# ! /bin/bash
+
+cmd=$( lftp -d sftp://mt49test@142.34.128.34 -p 22 -e 'set sftp:connect-program "ssh -o StrictHostKeyChecking=no -a -x -i ~/.ssh/bcmpcc"; mirror -e /outgoing temp ; quit;' )
+echo $cmd
+
+sleep 5
+date=$(date +%Y-%m-%d)
+mkdir $date
+mkdir $date/TDI17
+mkdir $date/TDI34
+
+sleep 3
+files=$( ls temp )
+
+shopt -s nocasematch
```

Comment:
Grabs the files, separates by type, parses, and stores by date. We may want to cleanup the text files from the local bucket after saving the parsed file ? 

Also sorting by file date, rather than the date the script runs, would be ideal. Although, it might be worthwhile to include the first line of the TDI's and handle the date stuff in the reconciliation program?

Reply:
I think as for the local stuff is concerned it is okay to be anyways until we get more structure. 

we can skip the date nesting all together and just put the files under the types